### ISSUE TITLE
Fix bug in graph editor where if you rename variable, its indicent edges disappear (but reappear if you close and reopen the graph editor)

### ISSUE THREAD

**Issue by jdramsey** [cmu-phil/tetrad#1693]

**Comment by jdramsey** [comment]
Fixed in branch.

**Comment by cg09** [comment]
Thank you. Now if only we could draw edges from a graph box after renaming
variables. Maybe we can?

**Comment by jdramsey** [comment]
Nope that doesn't work yet. I'll re-open this.

**Comment by jdramsey** [comment]
Sorry my brain's turned to jelly. I'll do this later.

**Comment by jdramsey** [comment]
That turned out to be trickier than I thought. Eventually, I realized how I had solved that before and did it. I need to think about it, though. Note to self: It turns out the code needs to identify graph nodes with the same name. If you use object-identity, the updaters break. The solution was if you have a null node in the edgeLists map, the thing to do is to reconstitute the map since a node of that name is already there and it just needs to be mapped properly.
